src/test/java/org/cactoos/list/ListOfTest.java

```diff
@@ -109,9 +108,9 @@ public void sensesChangesInIterable() throws Exception {
             () -> Collections.nCopies(size.incrementAndGet(), 0).iterator()
         );
         MatcherAssert.assertThat(
-            "Can't sense the changes in the underlying iterable",
+            "Can sense the changes in the underlying iterable",
             list.size(),
-            new IsNot<>(new IsEqual<>(list.size()))
+            new IsEqual<>(list.size())
```

Comment:
@iakunin this test is very strange, what is it testing? I think it was relate to ListEnvelope, but it's not the case anymore, we should remove it IMHO. @paulodamaso what do you think?

Reply:
Removed